### ISSUE TITLE
crash when doing torusctl wipe

### ISSUE THREAD

**Issue by sokoow** [coreos/torus#241]
```
This will wipe all metadata for torus.
Please type `YES`, all caps to confirm: 
YES
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x48dd1d]

goroutine 1 [running]:
panic(0xb397c0, 0xc820010110)
	/usr/lib/go-1.6/src/runtime/panic.go:464 +0x3e6
github.com/coreos/torus.WipeMDS(0xc288d8, 0x4, 0x0, 0x0, 0x0, 0x7ffe84e548d6, 0x11, 0x0, 0x0, 0x0, ...)
	/opt/go/src/github.com/coreos/torus/metadata.go:142 +0x1dd
main.wipeAction(0x102a620, 0xc820232480, 0x0, 0x2)
	/opt/go/src/github.com/coreos/torus/cmd/torusctl/wipe.go:42 +0x2f0
github.com/coreos/torus/vendor/github.com/spf13/cobra.(*Command).execute(0x102a620, 0xc8202322e0, 0x2, 0x2, 0x0, 0x0)
	/opt/go/src/github.com/coreos/torus/vendor/github.com/spf13/cobra/command.go:565 +0x85a
github.com/coreos/torus/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x1029c20, 0x102a620, 0x0, 0x0)
	/opt/go/src/github.com/coreos/torus/vendor/github.com/spf13/cobra/command.go:651 +0x55c
github.com/coreos/torus/vendor/github.com/spf13/cobra.(*Command).Execute(0x1029c20, 0x0, 0x0)
	/opt/go/src/github.com/coreos/torus/vendor/github.com/spf13/cobra/command.go:610 +0x2d
main.main()
	/opt/go/src/github.com/coreos/torus/cmd/torusctl/torusctl.go:47 +0x30
```

**Comment by sgotti** [comment]
PR #239

**Comment by sgotti** [comment]
This should be closed since #239 has been merged.

**Comment by barakmich** [comment]
Closed for #239
